### PR TITLE
Travis test config and virtualenv naming 

### PR DESCRIPTION
**Pull request by pmackay**
This PR handles both because I setup the Travis testing on top of the changes for virtualenv package naming (to test it!). Please review and comment when you can. If they need splitting up I can look at that.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+sudo: required
+dist: trusty
```
Comment (thefinn93) [inline .travis.yml]: Shouldn't we at least use the latest LTS? TBH i'd prefer Debian Jessie...

Comment (thefinn93) [inline .travis.yml]: Unless there's some magical way to test multiple distros so we could verify that it installs the package when available and clones the git when not...

Comment (pmackay) [inline .travis.yml]: Travis uses 12.04 by default and 14.04 as beta. There's only a few choices re base distros to run on. More advanced distro testing is probably best done with Docker.

Comment (thefinn93) [inline .travis.yml]: Seriously? Damn that sucks. Whatever, this looks good enough.

**Comment by thefinn93** [comment]
Thanks for this! In the future if you plan to PR, i'd prefer each new feature be in a separate pull request, but this was small enough that I didn't really care.

**Pull request merged**